Libraries/LibJS/AST.h

```diff
@@ -49,7 +49,7 @@ class SharedFunctionInstanceData;
 
 template<class T, class... Args>
 static inline NonnullRefPtr<T>
-create_ast_node(SourceRange range, Args&&... args)
+create_ast_node(SourceRange const& range, Args&&... args)
 {
     return adopt_ref(*new T(move(range), forward<Args>(args)...));
```

Comment:
To this file in genral
it seem to take the source range by `const&` and then tries to move it?